compiler/AST/LoopExpr.cpp

```diff
@@ -169,6 +169,30 @@ GenRet LoopExpr::codegen() {
 }
 
 
+//
+// Remove the ref types and the autocopy functions for the types
+// that are about to be removed when removing 'block'.
+//
+// Todo: we probably want to do this for all types that have been
+// deleted, as a cleanup function at the end of resolve().
+//
+void removeDeadTypeStuff(BlockStmt* block) {
```

Comment:
perhaps removeTypesAndCopyFnsFromDeadBlock ?

Reply:
Actually I think this is short enough to simply include it inside of removeTypeBlocks.